include/fmt/format.h

```diff
@@ -3484,9 +3484,18 @@ template <typename Char> struct udl_arg {
   }
 };
 
+// Converts string literals to basic_string_view.
 template <typename Char, size_t N>
-FMT_CONSTEXPR basic_string_view<Char> literal_to_view(const Char (&s)[N]) {
-  return {s, N - 1};
+FMT_CONSTEXPR basic_string_view<Char> fmt_string_to_view(const Char (&s)[N]) {
```

Comment:
`fmt_string` is a bit ambiguous. I suggest renaming `fmt_string_to_view` to `compile_string_to_view`, since it's for converting `fmt::compile_string` subclass object into a string view.